### ISSUE TITLE
Q about few-shot action recognition

### ISSUE THREAD

**Issue by Liu-arch** [Anirudh257/strm#32]
Hello, I have a principle question. Although in this paper we use N-way K-shot for training, we don't have actions from the training set in our test set. Can this still be called a FEW SHOT problem? Isn't this a ZERO-shot problem?
